.snippets/code/develop/blockchains/testing/setup/integration-testing-module.rs

```diff
@@ -0,0 +1,4 @@
+pub mod integration_testing {
```

Comment:
I'm assuming here you're doing what `mock.rs` does? https://github.com/paritytech/polkadot-sdk-parachain-template/blob/master/pallets/template/src/mock.rs

Reply:
well, `mock.rs` was covered on the` pallet testing` page. I was testing the real Runtime instead of the mocked one for this page. I based this page on the following code: https://github.com/AstarNetwork/Astar/tree/master/tests/integration

And then, my humble implementation: https://github.com/nhussein11/solochain-template 

WDYT @CrackTheCode016 ?

Reply:
(btw, this is the closest thing I found to an `integration test` tbh)

Reply:
Ah, I see! I personally have never done/experienced this, let me look a little bit at those implementations and re-review with this in mind. Apologies!